### ISSUE TITLE
Add an option to toggle filename truncation from the listing view

### ISSUE THREAD

**Issue by alyssadev** [x0b/rcx#116]
#### Pre-Submission checklist

 - [x] There is no existing issue that already asks for this feature.<br />
       _If no, please add a_ +1, _and maybe a comment if you have more to add._
 - [x] The feature already exists in rclone (on your PC or in Termux) (rclone does not truncate filenames when listing directories)
 - [x] I am prepared to help make this feature

#### What version of RCX are you using (About -> App version)?
1.11.4

#### What problem are you trying to solve?
On my phone screen in portrait mode, filenames that are longer than would fit in the screen have the middle truncated. This results in file listings where each item appears identical except for the file size, causing confusion when trying to select the correct file.
![RCX file listing](https://user-images.githubusercontent.com/63546997/110046304-bde62b80-7d86-11eb-9d74-5348f09ffbe9.jpg)

#### What should RCX be able to do differently with help this problem?
If possible, a toggle could be added to turn off this truncation, or move the truncation to the end of the filename instead, whichever the user prefers. Rather than "abcdefgh...tuvwxyz", an alternative display could be "abcdefghijklmno...". I've had a poke through the source but I couldn't find where filename truncation occurs, I assume it's handed off to a library that handles the file listing. If that's the case, this would obviously depend on that library allowing this change in truncation.

**Comment by x0b** [comment]
You should be able to toggle between full and elided names in the top-level menu **⋮** > **Wrap filenames**. 

I didn't write that part, I'm not sure why its called that - maybe we should change the label to something like **Shorten filenames**?

**Comment by alyssadev** [comment]
I didn't even check that menu, didn't see Wrap Filenames at the bottom, I thought that menu was just for rclone commands e.g serve and link. Maybe if Wrap Filenames was moved into settings > look and feel, maybe some more quality of life customisation options in there as well, I think that would be a UX improvement. I'll see if I can draft a PR to help push that

**Comment by x0b** [comment]
Yeah, the menu seems to lack discoverability - I've also heard from people that didn't find the serve option there.

PRs are appreciated, but please keep the scope and length small - that makes it easier to review.

A few pointers:

- Add the UI elements here: https://github.com/x0b/rcx/blob/master/app/src/main/res/layout/look_and_feel_settings_fragment.xml
- Rig them up in the fragment class: https://github.com/x0b/rcx/blob/57ded5c680f9e41bf08a1be96d3a061681eca650/app/src/main/java/ca/pkay/rcloneexplorer/Settings/LookAndFeelSettingsFragment.java
- Add a non-translatable `pref_key_` and translatable UI strings here: https://github.com/x0b/rcx/blob/master/app/src/main/res/values/strings.xml

**Comment by alyssadev** [comment]
I've made a PR that moves the wrap filenames option into look and feel. I was going to keep the Wrap filenames option in the options menu where it is and just have it load the preference value, but I figured it was probably better to avoid confusion in duplicating options like that.

As for the other options, maybe rclone commands e.g link, serve can be moved into the left panel above the import/export options in a submenu, and i think it would be good UX to keep the currently hidden options e.g empty trash shown, so users know where to find them if they ever need them. If nobody's ever complained about it though, no point in making things more confusing.
